src/d3a/d3a_core/simulation.py

```diff
@@ -24,6 +24,9 @@
 import dill
 import click
 import platform
+import os
```

Comment:
Thanks a lot for that! Definitely not part of that story, but we can think about activating this https://www.python.org/dev/peps/pep-0008/#imports in flake, in order to support this:

Imports should be grouped in the following order:

    Standard library imports.
    Related third party imports.
    Local application/library specific imports.

No need to open a story yet, because my plan is to have a general discussion about code styling (essentially make our rules stricter), which will involve d3a-web too. We can discuss it then.